pkgs/applications/networking/mullvad/openvpn.nix

```diff
@@ -1,83 +1,108 @@
 { lib
+, stdenv
 , openvpn
 , fetchpatch
 , fetchurl
 , iproute2
-, autoconf
-, automake
+, libnl
+, autoreconfHook
+, pkg-config
 }:
 
 openvpn.overrideAttrs (oldAttrs:
   let
+    inherit (lib) optional;
```

Comment:
We are using this in two places where we could just have prefixed optional with lib..